compose-builder/.env

```diff
@@ -23,23 +23,23 @@ CORE_EDGEX_REPOSITORY=edgexfoundry
 APP_SVC_REPOSITORY=edgexfoundry
 DEVICE_SVC_REPOSITORY=edgexfoundry
 UI_REPOSITORY=edgexfoundry
-CORE_EDGEX_VERSION=3.1.0
-APP_SERVICE_CONFIG_VERSION=3.1.0
-APP_LLRP_VERSION=3.1.0
-APP_RECORD_REPLAY_VERSION=3.1.0
+CORE_EDGEX_VERSION=3.1.1
+APP_SERVICE_CONFIG_VERSION=3.1.1
+APP_LLRP_VERSION=3.1.1
+APP_RECORD_REPLAY_VERSION=3.1.1
 EDGEX_UI_VERSION=3.1.0
 DEVICE_BACNET_VERSION=3.1.0
-DEVICE_MODBUS_VERSION=3.1.0
-DEVICE_MQTT_VERSION=3.1.0
-DEVICE_REST_VERSION=3.1.0
-DEVICE_SNMP_VERSION=3.1.0
-DEVICE_VIRTUAL_VERSION=3.1.0
-DEVICE_LLRP_VERSION=3.1.0
+DEVICE_MODBUS_VERSION=3.1.1
+DEVICE_MQTT_VERSION=3.1.1
+DEVICE_REST_VERSION=3.1.1
+DEVICE_SNMP_VERSION=3.1.1
+DEVICE_VIRTUAL_VERSION=3.1.1
+DEVICE_LLRP_VERSION=3.1.1
 DEVICE_COAP_VERSION=3.1.0
-DEVICE_GPIO_VERSION=3.1.0
-DEVICE_UART_VERSION=3.1.0
-DEVICE_ONVIFCAM_VERSION=3.1.0
-DEVICE_USBCAM_VERSION=3.1.0
+DEVICE_GPIO_VERSION=3.1.1
+DEVICE_UART_VERSION=3.1.1
+DEVICE_ONVIFCAM_VERSION=3.1.1
+DEVICE_USBCAM_VERSION=3.1.1
 
 # Lock on Vault 1.14 (last MPL-2.0 version)
 VAULT_VERSION=1.14.5
```

Comment:
```suggestion
VAULT_VERSION=1.14.6
```

Reply:
please help check and update all other images to the lastest `patch` release

Reply:
Third party images are updated.